### ISSUE TITLE
dose it support umd way?

### ISSUE THREAD

**Issue by mlzboy** [mit-cml/workspace-multiselect#76]
### Check for duplicates

- [X] I have searched for similar issues before opening a new one.

### Description

https://codepen.io/mlzboy/pen/ZEdRdrw?editors=1010
write a snippet code with claude seems not work
![image](https://github.com/user-attachments/assets/80aae1b5-fcdd-4ca3-b6f2-9b63a7048bb3)

### Reproduction steps

1. 
2. 
3.

**Comment by HollowMan6** [comment]
No, it's an ESM module. I think it should be enough for most people and I don't have plans for UMD, but if you are interested, you are welcome to submit a PR and configure it as a dual ESM + UMD package.

**Comment by mlzboy** [comment]
I used this module in blockly 9 with umd support ...

**Comment by HollowMan6** [comment]
> I used this module old version in blockly 9 with umd support ... 

Oh really, that would be the intial version of this plugin, which I don’t remember I did anything specifically to support UMD. If you have any ideas for how to restore that support, feel free to send a PR.

> but after use this module
,workspace.getTopBlocks(true).length will not work as expect,when I drag a block from toolbox into workspace it will append to topblocks twice.but workspace.getAllBlocks(true).length works fine.so I want to check the new version dose any change for fix this issue

That sounds like a Blockly issue, maybe they have some bugs counting the blocks when we have customized IDraggable, I would suggest you to open an issue there.

**Comment by HollowMan6** [comment]
Close as a staling issue.
